marvel/main.go

```diff
@@ -11,7 +11,7 @@ import (
 	"strconv"
 	"time"
 
-	"github.com/joho/godotenv/cmd/godotenv"
+	"github.com/joho/godotenv"
 )
 
 type characterClient struct {
```

Comment:
It's just a Client, not a "character Client" alone. The same client could be used for more than fetching characters.